frontend/src/general/dialog/DialogTitle.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import PropTypes from "prop-types";
+import MuiDialogTitle from "@material-ui/core/DialogTitle";
+
+export default function DialogTitle(props) {
+  const { children } = props;
+  return (
+    <MuiDialogTitle id="delete-stream-dialog-title">{children}</MuiDialogTitle>
+  );
+}
```

Comment:
I feel like this component could also be inlined in the dialog component since it contains no logic, only a MuiDialogTitle component. Also, the ID should not be `delete-stream-dialog-title` for a generic component.

Reply:
k

Reply:
54c634e